src/main/java/seedu/address/logic/commands/IncludeCommand.java

```diff
@@ -0,0 +1,117 @@
+package seedu.address.logic.commands;
+
+import static java.util.Objects.requireNonNull;
+import static seedu.address.model.Model.PREDICATE_SHOW_ALL_PERSONS;
+
+import java.util.List;
+
+import seedu.address.commons.core.Messages;
+import seedu.address.commons.core.index.Index;
+import seedu.address.logic.commands.exceptions.CommandException;
+import seedu.address.model.Model;
+import seedu.address.model.ReadOnlyAddressBook;
+import seedu.address.model.person.Person;
+
+/**
+ * Includes a Social media to an existing person in uNivUSal.
+ */
+public class IncludeCommand extends Command {
+
+    enum Socials {
+        WHATSAPP,
+        TELEGRAM,
+        EMAIL,
+        INSTAGRAM
+    }
```

Comment:
I see that the Socials enum is declared in every class, perhaps it would be more convenient to make it a global constant 😃